src/Psalm/Internal/Analyzer/Statements/Block/TryAnalyzer.php

```diff
@@ -86,11 +86,12 @@ public static function analyze(
 
         $old_referenced_var_ids = $try_context->referenced_var_ids;
 
+        $previously_inside_try = $context->inside_try;
```

Comment:
Sorry, I'll nitpick, can you name it `was_inside_try` for consistency?

Reply:
This is the convention that has been used for every context flag like this

Reply:
Didn't notice we already had a convention there.